src/common/ChannelChatters.cpp

```diff
@@ -66,7 +66,12 @@ void ChannelChatters::addPartedUser(const QString &user)
 }
 void ChannelChatters::setChatters(UsernameSet &&set)
 {
-    *this->chatters_.access() = set;
+    this->chatters_.access()->merge(std::move(set));
+}
+
+void ChannelChatters::clearChatters()
```

Comment:
Remove this function since it's unused

Reply:
Oh yea, used them before last changes, forgot to clean it up